aten/src/ATen/templates/DispatchKeyNativeFunctions.h

```diff
@@ -1,4 +1,7 @@
 #pragma once
+
+// clang-format off
```

Comment:
can you just add a comment right above this, explaining the reasoning? That way someone sees it in the future and knows why to not get rid of it.

Reply:
thanks, will add it.